nullaway/src/main/java/com/uber/nullaway/dataflow/AccessPath.java

```diff
@@ -554,7 +565,14 @@ public MapKey getMapGetArg() {
 
   @Override
   public String toString() {
-    return "AccessPath{" + "root=" + root + ", elements=" + elements + '}';
+    return "AccessPath{"
+        + "root="
+        + root
```

Comment:
I assume string concatenation works with `null`, but a) did we test this?, b) would it make more sense to make this `root == null ? "this" : root`?

Reply:
Agreed, will make this change